src/commands/init.ts

```diff
@@ -68,29 +77,30 @@ export abstract class InitCommand extends BaseCommand {
     return parsed;
   }
 
-  async run(): Promise<void> {
-    const { flags } = await this.parse(InitCommand);
+  async doesDockerComposeYmlExist(): Promise<boolean> {
+    const files_in_current_dir = fs.readdirSync('.');
+    const default_compose = files_in_current_dir.some(f => f.includes('compose') && (f.endsWith('.yml') || f.endsWith('.yaml')));
+    return default_compose;
+  }
 
-    const from_path = await this.getComposeFromPath(flags);
+  async runProjectCreation(project_name: string): Promise<void> {
+    if (fs.existsSync(`./${project_name}`)) {
+      console.log(chalk.red(`The folder ./${project_name} already exists. Please choose a different project name or remove the folder`));
+      return;
+    }
+    this.log(`Start the process to create your project '${project_name}'.`);
+    const selections = await ProjectUtils.getSelections();
+    await ProjectUtils.downloadGitHubRepos(selections, project_name);
+    await ProjectUtils.updateArchitectYamls(this.app, selections, project_name);
+    await ProjectUtils.linkSelections(this.app, selections, project_name);
+    this.log(`Successfully created project ${project_name}.`);
```

Comment:
Can we make the call to action the last thing we log out?

```
Successfully created project test.

Now try running your project: `architect dev test/react/architect.yml`
```

Also we should color the command to make it stand out. `chalk.blue('architect dev test/react/architect.yml')`. Want to make it abundantly clear what the next step should be.

Reply:
That's a great idea. We write out a README file so I just point them to it. 
```
Successfully created project my-proj.
Now try running your project using the instructions in my-proj/README.md.
```